lua/music-controls/init.lua

```diff
@@ -18,26 +18,25 @@ end
 -- If not, it will display a notification.
 local function check_playerctl_installed()
     if vim.fn.executable('playerctl') == 0 then
-        vim.notify('playerctl is not installed', 'error')
+        notify('playerctl is not installed', 'error')
         return false
     end
 
     return true
 end
 
--- Execute the 'playerctl' command.
+-- Execute the 'playerctl' command and get it's output.
```

Comment:
Correct me if I'm wrong, but shouldn't it be:

```suggestion
-- Execute the 'playerctl' command and get its output.
```

Since `it's` is short for `it is`.

Reply:
Good catch... sometimes they get by me.